### PR TITLE
Update test matrix

### PR DESCRIPTION
**Pull request by westy92**
#### What is this PR doing?
Testing on supported Node.js versions.  The previous versions are no longer supported.
Reduce dependency strictness.

#### How should this be manually tested?
N/A

#### What are the relevant tickets?
#119

### CODE REVIEW

**Comment by westy92** [comment]
Here's the versions of Node.js that are currently supported: https://github.com/nodejs/LTS

**Comment by aiham** [comment]
@westy92 I'd prefer to know if they failed in 5 or 7 for some reason even if they're unsupported. If it wasn't so cheap to run I'd agree.

**Comment by westy92** [comment]
The Node.js 8 test failures seem to be [a known issue](https://github.com/node-nock/nock/issues/922).  Should we add test support for 8 once this is resolved?

**Comment by aiham** [comment]
@westy92 Thanks!

**Comment by aiham** [comment]
Yes, please remove 8.

File: test/opentok-test.js [hunk 1]
```diff
@@ -728,15 +728,15 @@ describe('#generateToken', function () {
     expect(defaultExpireToken).to.be.a('string');
     expect(helpers.verifyTokenSignature(defaultExpireToken, apiSecret)).to.be.true;
     decoded = helpers.decodeToken(defaultExpireToken);
-    expect(decoded.expire_time).to.be.within(inOneDay - delta, inOneDay + delta);
+    expect(parseInt(decoded.expire_time, 10)).to.be.within(inOneDay - delta, inOneDay + delta);
```
Comment (westy92) [inline test/opentok-test.js]: The `chai` update needed these to be `numbers`, not `strings`.

Comment (aiham) [inline test/opentok-test.js]: Sure thing.

**Comment by aiham** [comment]
Thanks @westy92 !

**Pull request merged**